mws/resource_mws_pas_test.go

```diff
@@ -0,0 +1,246 @@
+package mws
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databrickslabs/databricks-terraform/common"
+	"github.com/databrickslabs/databricks-terraform/internal/qa"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMWSPAS(t *testing.T) {
```

Comment:
rename to `TestMwsPAS`, otherwise matcher will not pick this test up in automation.

Reply:
Question: why do we have both patterns? TestMWSNetworks and TestMwsAccWorkspace

Reply:
Done